core/src/test/scala/unit/kafka/server/ReplicaAlterLogDirsThreadTest.scala

```diff
@@ -60,6 +61,7 @@ class ReplicaAlterLogDirsThreadTest {
 
     when(replicaManager.futureLogExists(t1p0)).thenReturn(false)
 
+    val time = new SystemTime()
```

Comment:
We should use `Time.SYSTEM`, not create a new instance.

Reply:
We should consider making `SystemTime` package private.

Reply:
Makes sense.